stroeer/fragment/stage/v1/stage_service.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
+
+import "stroeer/page/section/v1/section_page.proto";
+
+package stroeer.fragment.stage.v1;
```

Comment:
Late to the naming party: why `fragment.stage` package? Stages are usually part of a section and fragment seems unrelated for me. Are companions also stages?

Reply:
not quite correct, we need this very StageService within the ArticleDetailPage. Its neither `core` nor a `page`, that's why I suggested a `fragment` package. Not sure if we really need the `stage` package.

Reply:
A companion is just a name for a 'area' of content. This content could be (multiple) `stages`, `commercials` and `teaser`

Reply:
guess I would skip `stage` in the package path, but fine for me